corehq/apps/reports/app_config.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class ReportsModule(AppConfig):
+    name = 'corehq.apps.reports'
+
+    def ready(self):
+        from corehq.apps.reports import signals  # noqa
```

Comment:
don't think so, been the standard way to do it in django since `AppConfig`s were introduced in 1.7.

From docs:
"In practice, signal handlers are usually defined in a signals submodule of the application they relate to. Signal receivers are connected in the ready() method of your application configuration class. If you’re using the receiver() decorator, simply import the signals submodule inside ready()."